### DISCUSSION TITLE
loading multiple plugins with cypress-cloud

### DISCUSSION THREAD

**Discussion by jeremytjl** [currents-dev/cypress-currents-reporter#160] [Q&A]
in my cypress.config.js

setupNodeEvents (on, config) {
....
//Currently have this
require('./cypress/plugins/index.js')(on, config)
....
}

how do i integrate together with return cloudPlugin(on, config); ?? Using cypress 12.16.0 version

**Comment by agoldis** [comment]
https://github.com/currents-dev/cypress-cloud/discussions/153
